src/main/java/it/gov/pagopa/pu/bff/service/broker/BrokerServiceImpl.java

```diff
@@ -2,55 +2,47 @@
 
 import it.gov.pagopa.pu.bff.config.DefaultConfigFe;
 import it.gov.pagopa.pu.bff.connector.organization.client.BrokerEntityClient;
-import it.gov.pagopa.pu.bff.connector.organization.client.OrganizationSearchClient;
 import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
 import it.gov.pagopa.pu.bff.mapper.PersonalisationFE2ConfigFEMapper;
 import it.gov.pagopa.pu.p4pa_organization.dto.generated.Broker;
-import it.gov.pagopa.pu.p4pa_organization.dto.generated.Organization;
 import it.gov.pagopa.pu.p4paauth.dto.generated.UserInfo;
-import it.gov.pagopa.pu.p4paauth.dto.generated.UserOrganizationRoles;
-import org.apache.commons.lang3.StringUtils;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
-import java.util.Optional;
-
 @Service
+@Slf4j
 public class BrokerServiceImpl implements BrokerService {
 
-  private final OrganizationSearchClient organizationSearchClient;
   private final BrokerEntityClient brokerEntityClient;
   private final PersonalisationFE2ConfigFEMapper personalisationFE2ConfigFEMapper;
   private final DefaultConfigFe defaultConfigFe;
-  private final ConfigFE defaultFEConfig;
 
-  public BrokerServiceImpl(OrganizationSearchClient organizationSearchClient, BrokerEntityClient brokerEntityClient,
+  public BrokerServiceImpl(BrokerEntityClient brokerEntityClient,
                            DefaultConfigFe defaultConfigFe,
                            PersonalisationFE2ConfigFEMapper personalisationFE2ConfigFEMapper) {
-    this.organizationSearchClient = organizationSearchClient;
     this.brokerEntityClient = brokerEntityClient;
     this.defaultConfigFe = defaultConfigFe;
     this.personalisationFE2ConfigFEMapper = personalisationFE2ConfigFEMapper;
-    this.defaultFEConfig = getFEConfiguration(null);
   }
 
   @Override
   public ConfigFE getBrokerConfig(UserInfo user, String accessToken) {
-    String orgIpaCode = Optional.ofNullable(user.getOrganizationAccess())
-      .orElseGet(() -> user.getOrganizations().stream()
-        .findFirst()
-        .map(UserOrganizationRoles::getOrganizationIpaCode)
-        .orElse(null)
-      );
-
-    if (StringUtils.isNotBlank(orgIpaCode)) {
-      Organization organization = organizationSearchClient.getOrganizationByIpaCode(orgIpaCode, accessToken);
-      if (organization != null && organization.getBrokerId() != null) {
-        return getFEConfiguration(brokerEntityClient.getBrokerById(organization.getBrokerId(), accessToken));
-      }
+    if (user.getBrokerId() == null) {
+      log.warn("BrokerId is null, returning default configuration.");
+      ConfigFE defaultConfig = getFEConfiguration(null);
```

Comment:
let's store it on the constructor and always use the same instace (as it was before this changes)